src/main/java/com/jalasoft/PivotalAPI/RequestManager.java

```diff
@@ -0,0 +1,45 @@
+package com.jalasoft.PivotalAPI;
+
+import java.util.Map;
+
+import com.google.gson.JsonParser;
+import com.google.gson.JsonElement;
+import com.jayway.restassured.http.ContentType;
+import com.jayway.restassured.response.Response;
+import com.jayway.restassured.specification.RequestSpecification;
+import org.apache.log4j.Logger;
+
+
+public class RequestManager {
+    private static final RequestSpecification RESPONSE_SPECIFICATION = Authentication.getInstance ().getRequestSpecification ();
+
+    private static final Logger LOGGER = Logger.getLogger(RequestManager.class.getName());
+
+    private RequestManager () {
+    }
+
+    public static Response getRequest (String endpoint) {
+        return RESPONSE_SPECIFICATION.when ().get (endpoint);
+    }
+
+    public static Response postRequest (String endpoint, Map<String, Object> parameters) {
+        JsonParser jsonParser = new JsonParser ();
+        JsonElement jsonObject = jsonParser.parse(parameters.toString());
```

Comment:
Instead of converting to JsonElement you can send it directly like params
Please ask to the team